prep-css-syntax/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Syntax</title>
+  <style>
+    h1 {
+      font-family: Arial, Helvetica, sans-serif;
+    }
+    h2 {
+      font-style:: italic;
```

Comment:
Be sure to only include 1 colon `:` so that the h2 text becomes italic.

Reply:
![Screen Shot 2022-01-12 at 10 59 29 AM](https://user-images.githubusercontent.com/96929704/149204971-8b52fab9-d806-4460-8828-9927eca0b6f6.png)